fairworkflows/fairworkflow.py

```diff
@@ -414,6 +414,28 @@ def draw(self, filepath):
                 'Cannot produce visualization of RDF, you need to install '
                 'graphviz dependency https://graphviz.org/')
 
+    def publish_as_nanopub(self, use_test_server=False):
```

Comment:
I'm thinking that we should probably have a ```publish_all_steps``` kwarg here too that is ```True``` by default, as it has been raised in previous discussions that some users may not want to publish all steps (FAIR doesn't mean open, etc etc). In such a case a user may wish to control which they do and don't publish.

Reply:
I agree! How to solve that though? We need some way of referencing the step without it being published. A blank node saying 'unpublished step', or maybe there is something in the pplan ontology that we can refer to?

Reply:
Could maybe raise this with Tobias. A blank node with a 'unique' name (so that it gets tranformed by ```nanopub``` into PUBLISHED_NANOPUB_URI#blanknodename as usual) might be enough? Alternatively we can make this an issue of its own and come back to it.

Reply:
I've created #106 for this.